### PR TITLE
patternfly-bootstrap-treeview 

### PR DESCRIPTION
**Pull request by priley86**
- add our fork's reference to patternfly-bootstrap-treeview
- adding utility script to encourage consistency in shrinkwrap
- clean shrinkwrap after updating bootstrap-treeview dependencies in fork

### CODE REVIEW

**Comment by priley86** [comment]
holding this until [#3](https://github.com/patternfly/patternfly-bootstrap-treeview/pull/3) is ready

**Comment by dlabrecq** [comment]
LGTM. 

I always clear the npm and bower cache (in my scripts) prior to installing/shrink wrapping. A necessity when changing dependencies. Although, I would think the Travis user/environment should already be clean?

**Comment by priley86** [comment]
@dlabrecq i've updated this PR to reflect our fork of bootstrap-treeview and update our shrinkwrap. This should clean up our dependency warnings in gemnasium and satisfy our requirements for bootstrap-treview in [PTNFLY-1590](https://patternfly.atlassian.net/browse/PTNFLY-1590).

I added the shrinkwrap script for utility purposes, and because I noted that generating with npm v2 and devDependencies installed can lead to problems. Are you thinking we could introduce this into our Travis pipeline? Maybe we can discuss this with the group for future.

**Comment by priley86** [comment]
@dlabrecq this should be ready after review :bowtie:

File: tests/pages/wizard.html [hunk 1]
```diff
@@ -3,7 +3,7 @@
 layout: page
 title: Wizard
 resource: true
-url-js-extra: '//rawgit.com/jonmiles/bootstrap-treeview/v1.2.0/dist/bootstrap-treeview.min.js'
+url-js-extra: '//rawgit.com/patternfly/patternfly-bootstrap-treeview/v1.0.0/dist/bootstrap-treeview.min.js'
 ---
```
Comment (jeff-phillips-18) [inline tests/pages/wizard.html:6]: I believe the wizard no longer uses treeview so this can be removed completely.

**Comment by priley86** [comment]
@jeff-phillips-18 was just following the find/replace path as best I could. It looks like you are right - i've gone ahead and removed it from _wizard.html_.

**Comment by jeff-phillips-18** [comment]
Understood. Only caught it because I was involved.

LGTM 👍

**Comment by priley86** [comment]
@dlabrecq @jeff-phillips-18  thanks 👍

**Pull request merged**